doc/architecture/decisions/0010-cookies.md

```diff
@@ -0,0 +1,48 @@
+# 10. Cookies
+
+## Context 
+
+We should be clear about what we store in the session cookie and why. 
+
+We store a number of properties in the `session` cookie:
+
+| Property | Description |
+|----------|-------------|
+| eq-session-id | The session identifier |
```

Comment:
The word seperators should be consistent within the cookie - this seems to be the only property that uses `-`

Reply:
Fair point. This has been this way for a long time. Now seems like a good time to change it.